cssdata.py

```diff
@@ -0,0 +1,112 @@
+# import from existing libraries
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# import from my modules
+from dataimportingfunctions import get_data_dir_list
+
+def exp_num_list():
+    """"
+    define experiment numbers that you want to include from the whole data
+    """
+
+    exp_num_list = [7, 8, 9, 10]
+    return exp_num_list
+
+
+def relative_density():
+    """
+    define relative densities for each experiment and trials
+    """
+
+    dr_exp7 = np.array([55, 54, 47, 56, 52, 52, 39, 52, 41, 43,
+                        39, 39, 33, 35, 43, 35, 41, 44, 53]) / 100
+    dr_exp8 = np.array([76, 94, 89, 77, 72, 73, 85, 88, 90, 74]) / 100
+    dr_exp9 = np.array([41, 47, 47, 40, 39, 43, 41, 49, 41, 37,
+                        44, 47, 48, 44, 50, 47, 51, 48, 50, 50, 40, 44,
+                        44, 45, 41, 44, 43, 41, 43, 37, 42, 43, 36, 50,
+                        42, 45, 49, 43, 39, 43, 47, 48, 55, 44, 49, 41,
+                        55]) / 100
+    dr_exp10 = np.array([78, 72, 84, 79, 82, 86, 89, 74, 70, 74, 80, 78, 89,
+                         84, 89, 84, 81, 83, 91, 86, 78, 73, 72, 71, 76, 80, 73, 74, 73]) / 100
+    drs = [dr_exp7, dr_exp8, dr_exp9, dr_exp10]
+
+    return drs
+
+
+def to_dataframe(data_dir, exp_num_list=exp_num_list(), drs=relative_density()):
+    """make a list to contain dataframe for the whole data"""
+
+    df_all = []  # make an empty list to contain dataframes
+
+    # get the number of trials for each experiment
+    for exp in exp_num_list:
+        data_dir_list = get_data_dir_list(exp, basedir=data_dir)  # get dir list of trial files in each exp

```

Comment:
Why is this a list? Also don't add type to variable names. Remove the `get_` in the function name

Reply:
In addition, the reason that I used the variable type to the variable was because I wanted to distinguish between the following:
- a list containing lists (e.g., [[1,2,3,4,5], [1,2,3,4,1], [3,4,1,2]]). In this case I would like to name it `datas_list`
- a list containing datas (e.g., [1,2,3,4,5]). In this case, I would like to name it `datas`
- a single variable. In this case, I would like to name it `data`

Is there other recommended naming convention?